.github/workflows/tests.yaml

```diff
@@ -127,6 +127,7 @@ jobs:
                       src/app/zap-templates/zcl/data-model/chip/microwave-oven-mode-cluster.xml \
                       src/app/zap-templates/zcl/data-model/chip/microwave-oven-control-cluster.xml \
                       src/app/zap-templates/zcl/data-model/chip/door-lock-cluster.xml \
+                      src/app/zap-templates/zcl/data-model/chip/ecosystem-information-cluster.xml \
```

Comment:
Blocking comment on waiting for CHIP-Specifications/connectedhomeip-spec#9499 to merge first